lib/src/pages/operador/operador_page.dart

```diff
@@ -21,7 +21,88 @@ class _OperadorPage extends StatelessWidget {
     });
 
     return Center(
-      child: Text('Tela de Operador'),
+      child: LoginPage(),
+    );
+  }
+}
+
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key? key}) : super(key: key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: SizedBox(
+        width: double.infinity,
+        height: double.infinity,
+        child: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Text(
+                'OPERADOR', style: TextStyle(fontSize: 30),
+              ),
+              SizedBox(height: 10),
+              TextField(
+                onChanged: (text){
+                },
+                keyboardType: TextInputType.number,
+                decoration: InputDecoration(
+                  labelText: 'Código',
+                  border: OutlineInputBorder(),
+                ),
+              ),
+              SizedBox(height: 10),
+              TextField(
+                onChanged: (text){
+                },
+                keyboardType: TextInputType.name,
+                decoration: InputDecoration(
+                  labelText: 'Nome',
+                  border: OutlineInputBorder(),
+                ),
+              ),
+              SizedBox(height: 10),
+              TextField(
+                inputFormatters: [
+                  // obrigatório
+                  FilteringTextInputFormatter.digitsOnly,
+                  CpfInputFormatter()
```

Comment:
Essa classe `CpfInputFormatter` é importada de onde? Acredito que faltou o import dela no arquivo

Se for do próprio Flutter, desconsidere esse comentário